_targets/ia32/generic/preinit.plo.yaml

```diff
@@ -9,10 +9,7 @@ contents:
   - phfs hd1 2.2 raw
   - phfs hd2 2.3 raw
   - phfs hd3 2.4 raw
-  - if: '{{ env.CONSOLE == "serial" }}'
-    str: console 0.0
-  - if: '{{ env.CONSOLE != "serial" }}'
-    str: console 3.0
+  - console {{ env.CONSOLE_DEVICE }} {{ env.CONSOLE_DEVICE2 }}
```

Comment:
Notice that `console 0.0 0.0` is possible but it probably doesn't make sense. Consider using a single `CONSOLE_DEVICES` variable and ensuring it contains only unique entries.